mava/utils/jax.py

```diff
@@ -42,3 +40,26 @@ def merge_leading_dims(x: chex.Array, num_dims: chex.Numeric) -> chex.Array:
 
     new_shape = (np.prod(x.shape[:num_dims]),) + x.shape[num_dims:]
     return x.reshape(new_shape)
+
+
+def convert_data(
```

Comment:
Need a more explicit name
```suggestion
def add_minibatch_dim(
```